### PR TITLE
Restructure build for individual pathogenesis manuscript

### PR DESCRIPTION
**Pull request by agitter**
### Description of the proposed additions or changes

This pull request will experiment with build script modifications.  The goal is to build a DOCX of only the pathogenesis content.

### Related issues

#732 #762

### Suggested reviewers (optional)

@rando2

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
The basic structure for building a docx of the pathogenesis manuscript is working locally.  I disabled AppVeyor builds temporarily.  Here is the docx generated by GitHub Actions: [pathogenesis-manuscript.docx](https://github.com/greenelab/covid19-review/files/5818721/pathogenesis-manuscript.docx)

I haven't tested anything with the author list or consortium members for the pathogenesis manuscript.  I'll clean up the broken svg images in the author list when I work on that.

**Comment by rando2** [comment]
This looks fantastic, @agitter. I'm actually going to email out the docx to everyone so they have a almost-final version! Would it be helpful if I started updating the metadata.yaml to reflect who is on this author list?

**Comment by agitter** [comment]
Let's wait until I can test a couple ideas for how to add the paper-specific author information to the metadata file.  Hopefully I can work on it more later today.

**Comment by rando2** [comment]
That sounds great @agitter! I'm happy to do whatever I can to help -- thank you so much for everything you've already done on this, it looks awesome and will really help with running things by people!

**Comment by rando2** [comment]
Hi @agitter, I am totally prepared to add in the roles for each manuscript as we finalize them on each manuscript hub issue (e.g., #744). Honestly this will be a great way for me to keep track of everything while going through the commit histories!

I think @cgreene had suggested using a random element to this one like in the Deep Review (so people in category 6 are randomized)-- would that be hard to switch out?

I think for the merged HTML document, intuitively I would say to just use the set (in the python sense of de-duplicated collection) of contributions. 

I think that addresses your questions, I might try opening a PR later today to test my ability to assign per-document author contributions!

**Comment by cgreene** [comment]
I think I was probably overly complicating things with the randomization. I'm happy to drop that suggestion in favor of simplicity + marking contribution categories.

**Comment by agitter** [comment]
Now that I'm setting the author order in a Python script anyway, randomization isn't necessarily too complicated.  Randomly re-ordering every build like deep review v2 would be fairly easy.  Randomly ordering once and then locking it would take more work, but I won't rule it out of that's what you (@rando2 and @cgreene) think is fair to the authors.

@rando2 I'll make a few more changes to support the individual manuscript contributions, and then this may be ready to review.  Because I edited the metadata file heavily, I'd like to merge this before you make any changes to it.

Taking the union of contributions should be possible.  I'll play around with that.

**Comment by agitter** [comment]
@rando2 I added support for setting the contributions in the merged manuscript to be the union of contributions from the individual manuscripts.  If contributions are not set for an author, they will be stored as **MISSING** and print a warning in the build log.  I added example contributions for you in the metadata to demonstrate the behavior.

If this build passes, it could be a good time to add some usage documentation and request a review.  We can work to merge this so you can start adding the real metadata for the pathogenesis manuscript.  I could work on author ordering more or listing the consortium members in a separate pull request.

**Comment by agitter** [comment]
This is ready for review.  I added some documentation and support for the consortium author.  Now there is an author "COVID-19 Review Consortium" in the metadata file.  This author should be listed and ordered in each individual manuscript.  All authors except COVID-19 Review Consortium and the Immunology Institute of the Icahn School of Medicine are listed under the author list as consortium members.  Feel free to modify the consortium name, how the members are listed in the front matter, etc.

I also modified contributions in the merged manuscript so that authors who are not yet listed as authors of any individual manuscript can use their existing general contributions.

File: build/README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ Format conversion is done using [Pandoc](https://pandoc.org/MANUAL.html).
 Each file specifies a set of pandoc `--defaults` options for a given format.
 To change the options, either edit the YAML files directly or add additional `--defaults` files.
 
+If the `BUILD_HTML` or `BUILD_PDF` environment variables are not false or the `BUILD_DOCX` environment variable is set to `true`, the build script will produce a merged manuscript
+with content from all individual manuscripts.
```
Comment (rando2) [inline build/README.md:20]:
```suggestion
If the `BUILD_HTML` or `BUILD_PDF` environment variables are not false or the `BUILD_DOCX` environment variable is set to `true`, the build script will produce a merged manuscript with content from all individual manuscripts.
```

**Review by rando2 — APPROVED** [review]
Amazing Tony! I'll merge this in and then go back and update the metadata for the authors, if that sounds good (to reflect the author order for pathogenesis and to move anyone who is not on pathogenesis to 1 paper that is a good fit for them!)

**Pull request merged**